frappe/client.py

```diff
@@ -424,4 +424,21 @@ def validate_link(doctype: str, docname: str):
 			frappe.PermissionError
 		)
 
-	return frappe.db.get_value(doctype, docname, cache=True)
+	values = frappe._dict()
+	values.name = frappe.db.get_value(doctype, docname, cache=True)
+
+	fields = frappe.parse_json(fields)
+	if not values.name or not fields:
+		return values
+
+	try:
+		values.update(get_value(doctype, fields, docname))
+	except frappe.PermissionError:
+		frappe.clear_last_message()
+		frappe.msgprint(
+			_("You need Read permissions to fetch values from {} {}")
+			.format(frappe.bold(doctype), frappe.bold(docname)),
```

Comment:
```suggestion
			_("You need {} permission to fetch values from {} {}")
			.format(frappe.bold("Read"), frappe.bold(doctype), frappe.bold(docname)),
```

Reply:
Permission names are translatable AFAIK. Maybe I can just do something like this?
```suggestion
			_("You need <strong>Read</strong> permission to fetch values from {} {}")
			.format(frappe.bold(doctype), frappe.bold(docname)),
```

Reply:
Alternatively, I can transalate **Read** separately..